events/models.py

```
@@ -342,7 +342,9 @@ def name_with_translations(self):
 
     @property
     def is_external_ticket_system_event(self):
-        return self.ticket_system in list(zip(*self.EXTERNAL_TICKET_SYSTEM_CHOICES))[0]
+        return self.ticket_system in next(
+            iter(zip(*self.EXTERNAL_TICKET_SYSTEM_CHOICES))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)